Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Scenes/Login.cs

```diff
@@ -58,7 +58,16 @@ protected async Task TryLogin()
 
             LoginSceneIndex = SceneManager.GetActiveScene().buildIndex;
 
-            SceneManager.LoadScene(MainSceneName);
+            // Attempt scene transition based on the login object sceneToLoad value
+            try
+            {
+                SceneManager.LoadScene(sceneToLoad);
+            }
+            catch (Exception e)
+            {
+                Debug.LogError(e);
+                throw;
+            }
```

Comment:
This try catch block is still throwing, seems redundant.

Reply:
removed